### DISCUSSION TITLE
TUI for risingwave

### DISCUSSION THREAD

**Discussion by zen-xu** [risingwavelabs/risingwave#16250] [General]
Hello everyone, I have developed a risingwave adapter for Harlequin.

[Harlequin](https://harlequin.sh) is **An easy, fast, and beautiful database client for the terminal.** However, since RisingWave has not fully implemented PostgreSQL, there are some compatibility issues. To address this, I have developed [harlequin-risingwave](https://github.com/zen-xu/harlequin-risingwave).

![image](https://github.com/risingwavelabs/risingwave/assets/38552291/40c8903f-c845-4622-b947-a9b612c2ce7c)

**Comment by Pandas886** [comment]
I think it's better to implement a web UI.
